setup.py

```diff
@@ -7,9 +7,9 @@
 with open(convert_path('src/version.py')) as f:
     exec(f.read(), version)
 
-replacePackagePath = partial(re.compile(rf'^src').sub, 'rsstt')
+replacePackagePath = partial(re.compile('^src').sub, 'rsstt')
```

Comment:
```suggestion
replacePackagePath = partial(re.compile(r'^src').sub, 'rsstt')
```

It is a common custom to make any regex strings r-string, even if unnecessary.